micromagneticmodel/driver.py

```diff
@@ -325,3 +343,19 @@ def _setup_working_directory(system, dirname, mode, append=True):
         workingdir = system_dir / f"{mode}-{next_number}"
         workingdir.mkdir(parents=True)
         return workingdir
+
+    @staticmethod
+    def _conversion_to_hms(time_difference):
+        total_seconds = math.ceil(time_difference.total_seconds())
+        hours, remainder = divmod(total_seconds, 3600)
+        minutes, seconds = divmod(remainder, 60)
+        return f"{hours:02}:{minutes:02}:{seconds:02}"
+
+    def _update_info_json(self, start_time, end_time, success):
```

Comment:
Do we need to pass `start_time` or should this be read from the json file as it is already written in there?

Reply:
You are right, we don't have to pass `start_time` but it might be easier to to that after all as we anyway have access to that variable in the single place where we want to call the update function (then we don't have to parse the date string).